doc/develop/tools/clion.rst

```diff
@@ -0,0 +1,270 @@
+
+
+Create a CMake presets file
+***************************
+
+This presets_ file will activate the Python virtual environment
+and will define the target board for CMake builds.
+
+.. _presets: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html
+
+You can find more information on how CLion works with CMake presets on
+`this web help page`_.
+
+.. _this web help page: https://www.jetbrains.com/help/clion/cmake-presets.html
+
+#. In the **Project** view, locate the
+    ``zephyrproject\zephyr\samples\hello_world`` folder.
+
+#. Right-click the ``hello_world`` folder and select **New | File**.
+
+#. Name the file ``CMakeUserPresets.json``.
+
+#. Add the following code into it:
+
+   .. code-block::
+      :emphasize-lines: 13,14
+      {
+         "version": 3,
+            "cmakeMinimumRequired": {
+            "major": 3,
+            "minor": 24
+         },
+         "configurePresets": [
+            {
+               "name": "Zephyr-Debug",
+               "generator": "Ninja",
+               "binaryDir": "build",
+               "cacheVariables": {
+                  "BOARD": "nucleo_f302r8",
+                  "CMAKE_PROGRAM_PATH": "<openocd_installation>;<home>/zephyrproject/.venv/Scripts"
+               }
+            }
+         ],
+         "buildPresets": [
+            {
+               "name": "Zephyr-Debug",
+               "configurePreset": "Zephyr-Debug"
+            }
+         ]
+      }
+
+#. Customize the following lines:
+
+   * ``"BOARD":...`` - specify the board you are using.
+
+   * ``"CMAKE_PROGRAM_PATH":...`` - set the path to Zephyr environment scripts
+     (delete this part if you are not using a virtual environment) and set
+     the path to your flashing software (OpenOCD in our example).
+
+#. Save the file. Make sure it is located in the ``hello_world`` folder,
+    i.e. the same folder as your CMake project.
+
+Load CMake
+**********
+
+#. In the **Project** tree, locate the ``CMakeLists.txt`` file in the
+    ``hello_world`` folder.
+
+#. Right-click it and select **Load CMake Project**.
+
+   CMake load should finish successfully.
+
+Configure CMake profiles
+************************
+
+CLion automatically creates a CMake profile for any detected preset,
+but such profiles are disabled by default. Follow the steps below to
+enable the profile associated with our preset and set it as a working
+profile for the project.
+
+#. Go to **Settings | Build, Execution, Deployment | CMake**.
+
+#. Select the **Zephyr-Debug preset** profile and set the **Enable profile**
+   checkbox.
+
+#. Delete the automatically created **Debug** profile. Alternatively,
+    move the **Zephyr-Debug preset** profile to the top of the list
+    to make it default for the project.
+
+#. Click **Apply** and close the dialog.
+
+   CMake project will be reloaded.
+
+Build the project
+*****************
+
+#. Check the top toolbar: the **Zephyr-Debug** profile and the
+   **zephyr_final** configuration should be selected.
```

Comment:
actually I don't have zephyr_final, I only have zephyr and zephyr_pre0 (and bunch of other of course, but no zephyr_final)
but I altered these lines: 
```
        "BOARD": "native_posix_64",
        "CMAKE_PROGRAM_PATH": "<home>/zephyrproject/.venv/Scripts"
```
since I don't use openOCD for posix or nRF53